nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -117,7 +117,7 @@ spec:
 ----
 <1> Specifies the namespace of the object you want to scale.
 <2> Specifies that this trigger authentication uses a platform-native pod authentication method for authorization.
-<3> Specifies a pod identity. Supported values are `none`, `azure`, `aws-eks`, or `aws-kiam`. The default is `none`.
+<3> Specifies a pod identity. Supported values are `none`, `azure`, `gcp-gke`, `aws-eks`, or `aws-kiam`. The default is `none`.
```

Comment:
@xingxingxia Can you please help to confirm with this line?

Reply:
@xueli181114 @xingxingxia Is this line incorrect for OSD, or has it not just been added to OCP yet? I understand OCP documentation doesn't have it, but I'd like to have clarification on whether gcp-gke is not actually available, vs we have a gap in OCP documentation that also needs to be updated.

Reply:
@xueli181114 Sorry I'm not sure about it. Anyway, I took a look at OpenShift documentation [Understanding custom metrics autoscaler trigger authentications](https://docs.openshift.com/container-platform/4.15/nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.html), in the section "_Example trigger authentication with pod authentication providers_", it's said `Supported values are none, azure, aws-eks, or aws-kiam`. I guess you could confirm with Nodes QE.

Reply:
@sunilcio @asahay19 Can you help to check this?

Reply:
For the current version of Custom Metrics Autoscaler (2.12.1) these are the valid values:
```
none | azure | azure-workload | aws-eks | aws-kiam | gcp
```
(From https://keda.sh/docs/2.12/concepts/authentication/#re-use-credentials-and-delegate-auth-with-triggerauthentication)

Reply:
@joelsmith Thank you so much!